### PR TITLE
Update pin for kealib

### PR DESCRIPTION
**Pull request by gillins**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Is this correct @ocefpaf? I haven't done one of these before...

### CODE REVIEW

File: recipe/migrations/kealib1413.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+__migrator:
+  build_number: 1
+  kind: version
+  migration_number: 1
+kealib:
+- 1.4.13
```
Comment (xhochy) [inline recipe/migrations/kealib1413.yaml:6]: `run_exports` in on `x.x` so this doesn't seem to need a migration. You could just change the entry in `recipe/conda_build_config.yaml` to `1.4`

**Comment by CJ-Wright** [comment]
I'm confused, why does this need to be repinned to a new micro version number?

File: recipe/conda_build_config.yaml [hunk 2]
```diff
@@ -435,7 +435,7 @@ json_c:
 jsoncpp:
   - 1.8.4
 kealib:
-  - 1.4.10
+  - 1.4.13
```
Comment (isuruf) [inline recipe/conda_build_config.yaml:438]:
```suggestion
  - 1.4
```

**Comment by CJ-Wright** [comment]
Sorry I should have been more specific. It seems that the package hasn't broken its ABI, at least according to the current pinning range. If that pinning range (defined by max_pin) is wrong then we should fix that otherwise do we need this update? (This might be a question for other members of core, there could be a subtlety that I'm missing)

**Comment by CJ-Wright** [comment]
I'll update the bot to issue pins at the max_pin level

**Comment by beckermr** [comment]
The bot didn’t issue this pr fwiw.

**Comment by CJ-Wright** [comment]
Yes, I know, but the bot has issued PRs down to the micro version when only minor was needed. https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/517/files

**Pull request merged**